src/ILOPool.sol

```diff
@@ -442,6 +421,7 @@ contract ILOPool is
             uint256 amount0Left, 
             uint256 amount1Left
         ) {
+        uint256 BPS = 10000;
```

Comment:
Any reason we removed the BPS constant? It appeared too many times in this file already